tox.ini

```diff
@@ -32,3 +32,6 @@ exclude = .eggs,.tox
 
 [pytest]
 testpaths = testing
+markers =
+    hookwrapper
+    optionalhook
```

Comment:
@davehunt I can't seem to find these being used (read) anywhere? 🤔

Reply:
I don't recall why these were needed, perhaps they aren't necessary anymore. If the tests are passing, then I would assume it's safe to remove these.

Reply:
Ok, I'll remove them in a separate PR for traceability then. 👍